modyn/trainer_server/internal/dataset/local_dataset_handler.py

```diff
@@ -0,0 +1,121 @@
+import os
+from typing import Optional
+
+import numpy as np
+from modyn.common.trigger_sample.trigger_sample_storage import TriggerSampleStorage
+
+LOCAL_STORAGE_FOLDER = ".tmp_offline_dataset"
+
+
+class LocalDatasetHandler(TriggerSampleStorage):
+    """
+    Class that wraps TriggerSampleStorage to use it as a local storage for samples.
```

Comment:
The current class doc string could also describe what `TriggerSampleStorage` is doing. Could we clearly state what this class does compared to TriggerSampleStorage?